Fix for libgit2->=1.8.0:
warning: call to undeclared function 'git_error_set'

Index: src/blob.c
--- src/blob.c.orig
+++ src/blob.c
@@ -28,6 +28,7 @@
 #define PY_SSIZE_T_CLEAN
 #include <Python.h>
 #include <git2.h>
+#include <git2/sys/errors.h>
 #include "diff.h"
 #include "error.h"
 #include "object.h"
